server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationDao.java

```diff
@@ -113,6 +113,10 @@ public void setDefaultGroupId(DbSession dbSession, String uuid, GroupDto default
     getMapper(dbSession).updateDefaultGroupId(uuid, requireNonNull(defaultGroupId, "Default group id cannot be null"), system2.now());
   }
 
+  public boolean getNewProjectPrivate(DbSession dbSession, OrganizationDto organization) {
```

Comment:
`OrganizationDao#getNewPrivateProject` I just don't understand what this method is supposed to do.

what about "isNewProjectPrivateByDefault" ?